.gitignore

```diff
@@ -10,4 +10,8 @@ __pycache__/
 *.pyc
 
 # Logs
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+
+# Env
+.env
+venv/
```

Comment:
AD, let's agree on a name that we can use for all of them (I liked `kvasir-env`, but open to changing)

Reply:
`kvasir-env` is good by me, I'll chance tonight